amundsen_application/static/js/components/AnnouncementPage/styles.scss

```diff
@@ -1,12 +1,16 @@
 // Copyright Contributors to the Amundsen project.
 // SPDX-License-Identifier: Apache-2.0
-
 @import 'variables';
+@import 'typography-default';
```

Comment:
We have to explicitly import to use the `@extend`?

Reply:
Yep, I was surprised at first too.

Not sure if we can fix it somehow!